app/HMS/Entities/Profile.php

```diff
@@ -70,6 +70,11 @@ class Profile
      */
     protected $dateOfBirth;
 
+    /**
+     * @var null|string
+     */
+    protected $discordUserId;
```

Comment:
this is actually the username+discriminator right? not the user_id snowflake

https://discord.com/developers/docs/resources/user#user-object-user-structure

To consider: 
should we change name from `discordUserId ` to `discordUsername` ? 
id say it should be `discordUsernameDiscriminator`  but there changing the soon anyway